server.ts

```diff
@@ -0,0 +1,65 @@
+import dotenv from "dotenv-flow";
+import express = require("express");
+import { Request, Response } from "express-serve-static-core";
+import bodyParser from "body-parser";
+import path from "path";
+import { RequestBuilder, Payload } from "yoti";
+import https from 'https';
+import fs from 'fs';
+import morgan from 'morgan';
+
+dotenv.config();
+
+const rawPort = process.env.SERVER_PORT || '5000';
```

Comment:
There are some people having issues with port 5000 (already used), I think it's a good moment to change it, what do you think? We can set 5001 for example.